src/main/java/com/fasterxml/jackson/databind/MapperFeature.java

```diff
@@ -661,7 +661,12 @@ public enum MapperFeature implements ConfigFeature
      *
      * @since 2.19
      */
-    REQUIRE_HANDLERS_FOR_JAVA8_OPTIONALS(true)
+    REQUIRE_HANDLERS_FOR_JAVA8_OPTIONALS(true),
+
+    /**
+     * @since 2.19
+     */
+    WRAP_EXCEPTIONS(true)
```

Comment:
Highly confusing since we already have `SerializationFeature.WRAP_EXCEPTIONS` and `DeserializationFeature.WRAP_EXCEPTIONS`...

Reply:
Right, will change.

Reply:
Modified to your original suggestion 👍🏼